src/app/bugs/components/bugs-menu/bugs-menu.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+    <div class="box box1"  [routerLink]="['/bugs/bugsdashboard']"><span>Severity</span></div>
```

Comment:
Lets use anchor tags here instead of div

Reply:
yes this updated with anchor tag

Reply:
yup anchor tag is updated, please check the branch code